docs/develop/power-automate-integration.md

```diff
@@ -50,35 +50,30 @@ When adding input parameters to a script's `main` function, consider the followi
 8. Objects must have their interface or class definition defined in the script. An object can also be defined anonymously inline, as in the following example:
 
     ```TypeScript
-    async function main(context: Excel.RequestContext): Promise<{name: string, email: string}>
+    function main(workbook: ExcelScript.Workbook): {name: string, email: string}
     ```
 
-9. Optional parameters are allowed and can be denoted as such by using the optional modifier `?` (for example, `async function main(context: Excel.RequestContext, Name?: string)`).
+9. Optional parameters are allowed and can be denoted as such by using the optional modifier `?` (for example, `function main(workbook: ExcelScript.Workbook, Name?: string)`).
```

Comment:
I'm conflicted because it looks weird in Power Automate (as other variables start with capital letters). @danielgwilson, what should our convention be here?